README/clientSecretBasic.md

```diff
@@ -0,0 +1,3 @@
+## Client Secret Basic
```

Comment:
I missed this on the previous one. Why do you create a readme file for each class? It doesn't contain any information specific to the class.

Reply:
The auth0 team wanted me to add a readme for each pull request stating the intended usage.